docs/whitepapers/shortpaper.mdx

```diff
@@ -0,0 +1,641 @@
+        </td>
+        <td>1
+        </td>
+        <td>1
+        </td>
+        <td>0.25
+        </td>
+        <td>62116000
+        </td>
+    </tr>
+    <tr>
+        <td>2024
+        </td>
+        <td>2
+        </td>
+        <td>0.5
+        </td>
+        <td>0.125
+        </td>
+        <td>61130500
+        </td>
+    </tr>
+    <tr>
+        <td>2025
+        </td>
+        <td>2
+        </td>
+        <td>0.5
+        </td>
+        <td>0.125
+        </td>
+        <td>60145000
+        </td>
+    </tr>
+    <tr>
+        <td>2026
+        </td>
+        <td>2
+        </td>
+        <td>0.5
+        </td>
+        <td>0.125
+        </td>
+        <td>59159500
+        </td>
+    </tr>
+    <tr>
+        <td>2027
+        </td>
+        <td>2
+        </td>
+        <td>0.5
+        </td>
+        <td>0.125
+        </td>
+        <td>58174000
+        </td>
+    </tr>
+    <tr>
+        <td>2028
+        </td>
+        <td>3
+        </td>
+        <td>0.25
+        </td>
+        <td>0.0625
+        </td>
+        <td>57681250
+        </td>
+    </tr>
+    <tr>
+        <td>2029
+        </td>
+        <td>3
+        </td>
+        <td>0.25
+        </td>
+        <td>0.0625
+        </td>
+        <td>57188500
+        </td>
+    </tr>
+    <tr>
+        <td>2030
+        </td>
+        <td>3
+        </td>
+        <td>0.25
+        </td>
+        <td>0.0625
+        </td>
+        <td>56695750
+        </td>
+    </tr>
+</table>
+Note: Calculations after 2030 are not shown for brevity.
+*Table 6: DataHighway Initial Token Issuance Halving Inflation Strategy Cycle*
+
+### Governance
+All DHX DAO decisions shall be made through on-chain governance once Sudo is removed, and will function much in the same way as [Polkadot Governance](https://wiki.polkadot.network/docs/en/learn-governance), with some exceptions.
+
+Participation in DataHighway Governance is required in order to take advantage of PoP mining. This participation takes the form of bonding DHX to a council member, or potential council member. This bonded DHX helps to ensure that all participants in the DataHighway work together to determine who should be on the council. If a council member is not chosen, and is not shortlisted, DHX tokens belonging to the participant that they bonded will be unbonded and returned to them automatically. They must then rebond those tokens to a different council member or candidate in order to continue PoP mining.
+
+##### Governance through MXC Supernodes a broker:
+Each MXC Supernode is viewed as a single identity on the DataHighway, thereby acting as a broker for all participants that have chosen them. When mining on an MXCSupernode, a portion of each participants’ mPower goes to a council chair, who then acts as a proxy on behalf of them when determining how an MXC Supernode will vote on a referendum. All council chairs will be required to vote on each referendum, otherwise the council chair benefit will be slashed.
```

Comment:
```suggestion
Each MXC Supernode is viewed as a single identity on the DataHighway, thereby acting as a broker for all participants that have chosen them. When mining on an MXC Supernode, a portion of each participants’ mPower goes to a council chair, who then acts as a proxy on behalf of them when determining how an MXC Supernode will vote on a referendum. All council chairs will be required to vote on each referendum, otherwise the council chair benefit will be slashed.
```